src/System.Diagnostics.StackTrace/src/project.json

```diff
@@ -5,7 +5,11 @@
         "Microsoft.TargetingPack.Private.CoreCLR": "1.1.0-beta-24415-03",
         "System.IO.FileSystem": "4.0.1",
         "System.Reflection.Metadata": "1.4.1-beta-24415-01",
-        "System.Collections.Immutable": "1.2.0"
+        "System.Collections.Immutable": "1.2.0",
+        "System.Runtime.InteropServices": {
+          "version": "4.1.0",
+          "exclude": "compile"
```

Comment:
That is what Wes recommended. This is still a lot like magic to me.  Instead of the project.json InteropServices, etc. like adding this to my csproj:

<ProjectReference Include="..\..\System.Collections\src\redist\System.Runtime.InteropServices.depproj">